style_editor.html

```diff
@@ -5386,6 +5393,18 @@
   return new BatteryLevelClass();
 }
 
+class VolumeLevelClass extends FUNCTION {
+  constructor() {
+    super("Returns 0-32768 based on volume level.", []);
+  }
+  run(blade) {}
+  getInteger(led) { return 0 + ((millis() * 7) & 0x7fff); }
```

Comment:
VolumeLevel wouldn't normally change this fast.
It would probably be better just return a single value.

Reply:
I cranked the speed up intentionally.  I suppose a static volume could suffice, but what if it's set to something like 75%, but you want to see what the result of the blade style on the tip looks like?
Do you think we could keep the movement if it's slowed down to like *4 or *5?

Reply:
Maybe we could make it something like:

((millis() / 500) % 11) * 32767 / 10

Reply:
Looks the same result to me. Sure?